<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_str">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Your string:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pe">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pe">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_equal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Operator +=</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Operator =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_size">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_eq">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_str1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_str2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ans">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>str1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>str2(if need)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>ans</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_memcpy">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>memcpy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_memmove">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>memmove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_strncpy">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>strncpy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_strcpy">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>strcpy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_memcmp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>memcmp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_strncat">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>strncat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_strcat">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>strcat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_strcmp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>strcmp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_strncmp">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>strncmp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="strxfrm">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>strxfrm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_strcoll">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>strcoll</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_strtok">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>strtok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_strlen">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>strlen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_strmemset">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>strmemset</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>350</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>n(if need)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
